oval/util.go

```diff
@@ -348,7 +348,7 @@ func isOvalDefAffected(def ovalmodels.Definition, req request, family string, ru
 
 		if running.Release != "" {
 			switch family {
-			case constant.RedHat, constant.CentOS, constant.Alma, constant.Rocky, constant.Oracle, constant.Fedora:
+			case constant.RedHat, constant.CentOS, constant.Alma, constant.Rocky, constant.Oracle:
```

Comment:
![image](https://user-images.githubusercontent.com/49754654/152708538-c558d0ec-7092-41e8-8876-77e237390d13.png)

`running.Release` and `ovalPack.Name` is not empty
`kernelRelatedPackNames` is neither empty(contains many package names)
but `ok` is always false.

Reply:
Since this may be the case, Fedora should also be considered.
https://github.com/future-architect/vuls/pull/541